### DISCUSSION TITLE
`crtl+shift+alt+t` can't rename a tab title

### DISCUSSION THREAD

**Discussion by GinoongFlores** [kovidgoyal/kitty#7285] [General]
Hello, newbie on kitty, I'm trying to rename a tab title, as it default its title to the path name of the directory I'm working, In tmux, I can do it by `crtl+b+,`. I read on the docs, and correct If I'm wrong if I use, the right command or not of  `crtl+shift+alt+t`. Yet, it creates another tab instead of setting a tab title, am I right? Thank you so much for understanding

**Comment by kovidgoyal** [comment]
You press it, then you type in your new tab title and press enter. ctrl+shift+t creates a new tab, ctrl+shift+alt+t renames the existing tab.

**Reply by GinoongFlores** [reply]
Hello @kovidgoyal, thanks for that. Yet, I'm quite confused, what do you mean by "pressing it"? the `crtl+shift+alt+t` seems creates another window for me. I look into this, closed issue https://github.com/kovidgoyal/kitty/issues/610. You say here, tabs will be rename based on the current directory. Although, it's not always a big deal imo, yet it's a good option.

**Comment by kovidgoyal** [comment]
On Sat, Mar 30, 2024 at 01:49:04AM -0700, Christian Paul Flores wrote:
> Hello @kovidgoyal, thanks for that. Yet, I'm quite confused, what do you mean by "pressing it"? the `crtl+shift+alt+t` seems creates another window for me. I look into this, closed issue https://github.com/kovidgoyal/kitty/issues/610. You say here, tabs will be rename based on the current directory. Although, it's not always a big deal imo, yet it's a good option.

The "new window" has

Enter the new title for this tab below. An empty title will cause the default title to be used.
>

So follow those instructions. If you arent seeing it, then make sure you
are running up-to-date kitty and start kitty with --config=NONE

**Reply by GinoongFlores** [reply]
Alright, thanks!, we'll look into it

**Reply by kumarUjjawal** [reply]
For macos you have do ctrl+shift+option+t
